fun/middleware/enforce_legal_acceptance.py

```diff
@@ -28,13 +30,9 @@
         r"""^/?$""",
     )
 )
-def get(self, key, default):
-    return getattr(self, key) if hasattr(self, key) else default
 
-LMS_BASE = get(settings, "LMS_BASE", "")
-PROT = get(settings, "HTTPS", "off") == "on" and "s" or ""
-LMS_REDIR = LMS_BASE and ("http%s://%s/" % (PROT, LMS_BASE)) or ""
-DEFAULT_AGREEMENT_FORM = "%s/payment/terms/" % (LMS_REDIR)
+TERMS_AND_CONDITIONS_AGREEMENT = '/payment/terms/'
+
 
 def terms_accepted(func):
```

Comment:
I think the name of this wrapper confusing, what about renaming it: `ensure_accepted_terms`? or something alike.